obs-visca-control.lua

```diff
@@ -264,6 +308,8 @@ function script_properties()
         create_camera_controls(props, camera_id, plugin_settings)
     end
     
+    local debug = obs.obs_properties_add_bool(props, "debug", "debug log (needs refresh)")
```

Comment:
When you register a callback, it will become effective immediately.
```suggestion
    local debug = obs.obs_properties_add_bool(props, "debug", "debug log (needs refresh)")
    obs.obs_property_set_modified_callback(debug, function (props, property, settings)
      ...
    end)
```